### ISSUE TITLE
XP from Experience Obelisk has changed breaking the integration with Pedestals

### ISSUE THREAD

**Issue by perlguy** [Mowmaster/Pedestals#254]
Experience Obelisk mod is changing things around, and no longer uses "raw_experience", but instead "cognitium".  "raw_experience" still exists, but looks like it's only there to keep other mods from breaking outright.  Playing DW20 1.20 pack, and Pedestal doesn't recognize "cognitium" as a valid source of XP, neither directly or via fluid converter.

For now, I used KubeJS to add the new recipe, and that worked, allowing me to make sculk via Pedestal/Catalyst block.
( Obelisk -> pipez fluid pipe -> pedestal+fluid converter upgrade <linked_to> pedestal+material generator upgrade over Sculk Catalyst block )

```
ServerEvents.recipes(event => {
    event.custom(
      {
        "type": "pedestals:fluidconverter",
        "inputFluidStack": {
          "fluid": "experienceobelisk:cognitium",
          "count": 250
        },
        "experience": {
          "experienceValue": 11
        },
        "conditions": [
          {
            "type": "forge:mod_loaded",
            "modid": "experienceobelisk"
          }
        ]
      }
    )
})
```

**Comment by Mowmaster** [comment]
https://github.com/Mowmaster/Pedestals/commit/f31840ff777f8acb4c8c5e8f96875b497dae2f88

**Comment by thelapisfox** [comment]
Just want to point out that this exact same issue is occurring in the same pack (Direwolf20's 1.20.1 pack) with Ender IO's Xp Obelisk.

**Comment by Mowmaster** [comment]
Yeah, that's what this post started as, I added a fix for 1.19 just haven't gotten around to fixing/updating 1.20 yet

**Comment by Mowmaster** [comment]
Okay, so both 
Ender io
And
Xp obelisk 
On 1.20 needs fixed.
Is ender io on 1.19?
